packages/front-end/components/Metrics/MetricsList.tsx

```diff
@@ -255,7 +255,7 @@ const MetricsList = (): React.ReactElement => {
     (m) => ({
       datasourceName: m.datasource
         ? getDatasourceById(m.datasource)?.name || "Unknown"
-        : "Manual",
+        : "None",
```

Comment:
nit: Should we have a warning in this case? Maybe a fast follow to help people clean up orphaned metrics.

Reply:
Yeah, a fast follow would be good